doc/reference/storage_dir.md

```diff
@@ -1,25 +1,38 @@
 (storage-dir)=
 # Directory - `dir`
 
- - While this backend is fully functional, it's also much slower than
-   all the others due to it having to unpack images or do instant copies of
-   instances, snapshots and images.
- - Quotas are supported with the directory backend when running on
-   either ext4 or XFS with project quotas enabled at the filesystem level.
+The directory storage driver is a basic backend that stores its data in a standard file and directory structure.
+This driver is quick to set up and allows inspecting the files directly on the disk, which can be convenient for testing.
+However, LXD operations are not optimized for this driver.
```

Comment:
A link to https://linuxcontainers.org/lxd/docs/master/reference/storage_drivers/#feature-comparison seems appropriate here to allow the reader to understand what "not optimized" means.

Reply:
And/or we move the next paragraph up to this section, as that explains it somewhat.